test/Base_Tests/src/Runtime/Stack_Traces_Spec.enso

```diff
@@ -21,6 +24,35 @@ deep_b = deep_c
 deep_a = deep_b
 
 add_specs suite_builder = suite_builder.group "Stack traces" group_builder->
+    group_builder.specify "avoid handler frames in stack" <|
+        lines = Vector.build b->
+            do_the_handling caught_panic =
+                st = caught_panic.stack_trace
+                st.each el->
+                    b.append el.to_display_text
+
+            my_exception_handler caught_panic =
+                do_the_handling caught_panic
+
+            f5 ~a = a
+            f4 ~a = f5 a...
+            f3 ~a =
+                Panic.catch Any handler=my_exception_handler <|
+                    f4 a...
+            f2 ~a = f3 a...
+            f1 ~a = f2 a...
+
+            f1 (Panic.throw My_Error.Error)
+
+        top_lines = lines.take (..While l-> l.contains "Vector.type" . not)
+        out = top_lines.join '\n'
+
+        forbidden1 = out.contains "my_exception_handler"
+        forbidden2 = out.contains "do_the_handling"
+
+        if forbidden1 || forbidden2 then
+            Test.fail 'Frames from exception handling should not be in the stacktrace:\n'+out
```

Comment:
this could be simplified to `out.should_not_contain "my_exception_handler"` etc.

Reply:
It is not easy to find such methods in VSCode editor right now. Fixed in a0f38545a8 - thank you.